### PR TITLE
#29  Add model file for bookmark table

### PR DESCRIPTION
**Pull request by oluwashane**
#### What does this PR do?

- Add model file for bookmark table

#### Description of Task to be completed?

- Add bookmark.js file to the model folder

- Define bookmark table

#### What is the GitHub issue ID for this task?
[finishes #29 ]

### CODE REVIEW

File: src/database/models/bookmark.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import mongoose from 'mongoose';
+const { model, Schema } = mongoose;
```
Comment (ebenezerdon) [inline src/database/models/bookmark.js:3]: Please add an empty line before `line 2`

File: src/database/models/bookmark.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    },
+    color_code: {
+        type: String,
+        required: true,
+    }
+});
+
+const bookMark = model('bookMark', bookMarkSchema);
+
+export default bookMark;
```
Comment (ebenezerdon) [inline src/database/models/bookmark.js]: Please add an empty line to indicate `End of File`

File: src/database/models/bookmark.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import mongoose from 'mongoose';
+const { model, Schema } = mongoose;
+
+const bookMarkSchema = new Schema({
+    tag_id: {
+        type: Number,
```
Comment (ebenezerdon) [inline src/database/models/bookmark.js]: Let's change `Number` to `Schema.Types.ObjectId`

Comment (oluwashane) [reply src/database/models/bookmark.js]: I've changed it.

**Review by ebenezerdon — APPROVED** [review]
LGTM

**Pull request merged**